README.md

```diff
@@ -116,20 +116,23 @@ You also need to install a few system dependencies: TeX, Poppler, and graphviz.
   ```
 
 > [!NOTE]
-> If your notebook submits hardware jobs to IBM Quantum, you must add it to the
-> list `notebooks-that-submit-jobs` in
-> [`scripts/nb-tester/notebooks.toml`](scripts/nb-tester/notebooks.toml). This
-> is not needed if the notebook only retrieves information.
->
+> When testing notebooks, we avoid sending jobs to IBM Quantum by patching
+> `least_busy` to return a fake backend. Try to make sure your notebook works
+> with this patch. To use the patch, add your notebook to the list
+> `notebooks_that_submit_jobs` in
+> [`scripts/nb-tester/notebooks.toml`](scripts/nb-tester/notebooks.toml). If
+> your notebook can't be tested with this patch, you must add it to the list
+> `notebooks_no_mock`.
```

Comment:
```suggestion
> If your notebook submits hardware jobs to Qiskit Runtime, you must add it to
> [`scripts/nb-tester/notebooks.toml`](scripts/nb-tester/notebooks.toml). If it
> can be run with simulators, i.e., the circuit is not too large, add it to `notebooks_that_submit_jobs`.
> Otherwise, add it to `notebooks_no_mock`.
```